MachineLearningPrototype/utils/preprocess_data.py

```diff
@@ -8,3 +8,9 @@ def remove_time_stamp(content):
     new_content = re.sub(r"\d{2}:\d{2}:\d{2}", "", content)
     new_content = re.sub(r"\[\d{4}-\d{2}-\d{2}.*?\]", "", new_content)
     return new_content
+
+def extract_quotation_content(content):
+    pattern = "\`\`\`([\s\S]*?)\`\`\`"
+    substring = re.findall(pattern, content)
+    result=''.join(substring)
```

Comment:
Instead of empty string '', we can use string with a space (' '), in case two quoted contents do not have line change or space in between.